cmd/ndm_daemonset/app/command/device-list.go

```diff
@@ -81,7 +81,6 @@ func NewSubCmdListBlockDevice() *cobra.Command {
 
 	// Bind & parse flags defined by external projects.
 	// e.g. This imports the golang/glog pkg flags into the cmd flagset
-	getCmd.Flags().AddGoFlagSet(goflag.CommandLine)
 	goflag.CommandLine.Parse([]string{})
 
 	getCmd.Flags().StringVar(&options.kubeconfig, "kubeconfig", "",
```

Comment:
The `kubeconfig` flag can also be removed in both `start.go` and `device-list.go`. This will be replaced by the `kubeconfig` flag from `controller-runtime` config.

Reply:
Addressed as part of https://github.com/openebs/node-disk-manager/pull/325/commits/3f441e29e61e631563790eb563a4d6920bd87f54